zipline/assets/assets.py

```diff
@@ -648,9 +648,10 @@ def _lookup_symbol_fuzzy(self, symbol, as_of_date):
 
         if not options:
             # no equity owned the fuzzy symbol on the date requested
-            SymbolNotFound(symbol=symbol)
+            raise SymbolNotFound(symbol=symbol)
+
 
-        if len(options) == 1:
+        if len(options) == 1 or len(set(map(itemgetter(0), options))) == 1:
```

Comment:
What do you think about `len({opt[0] for opt in options}) == 1` for pythonic clarity?